src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -210,6 +210,10 @@ PeerImp::send (Message::pointer const& m)
         // a small senq periodically
         large_sendq_ = 0;
     }
+    else if ((sendq_size % Tuning::sendQueueLogFreq) == 0)
+        JLOG (journal_.debug()) <<
```

Comment:
I'm getting the following warning from clang on this JLOG:
```
src/ripple/overlay/impl/PeerImp.cpp:214:9: warning: add explicit braces to avoid
      dangling else [-Wdangling-else]
        JLOG (journal_.debug()) <<
        ^
src/ripple/basics/Log.h:246:29: note: expanded from macro 'JLOG'
#define JLOG(x) if (!x) { } else x
```